src/lib/convertUnit.js

```diff
@@ -139,7 +147,7 @@ function convertUnit(value, sourceUnit, targetUnit, precision) {
   const converted = conversions[targetUnitNormalized][sourceUnitNormalized] * value;
 
   if (precision !== false) {
-    precision = Math.pow(10, parseInt(precision) || 5);
+    precision = Math.pow(10, Math.ceil(precision) || 5);
```

Comment:
`precision` should be a number here. If the user passed the wrong type, I don't think `parseInt` is going to produce a sensible result anyway, especially with no `radix` argument.